src/explorer.ts

```diff
@@ -61,7 +61,13 @@ export class KubernetesExplorer implements vscode.TreeDataProvider<KubernetesObj
 
     private async getClusters(): Promise<KubernetesObject[]> {
         const clusters = await kubectlUtils.getClusters(this.kubectl);
-        return clusters.map((cluster) => new KubernetesCluster(cluster.name, cluster));
+        return clusters.map((cluster): KubernetesCluster => {
+            // TODO: this is slightly hacky...
+            if (cluster.name == 'minikube') {
```

Comment:
maybe we should map this to an enum so we can perform a switch on this, and modify `kubectlUtils.getClusters` to drop in some metadata about cluster type into its return value

Reply:
I didn't quite follow this suggestion.